src/styles/reset.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+}
+
+p {
```

Comment:
You don't need to reset p tag with margin: 0 if you already reseted the body with the same style! Remember, CSS is a "cascade", the parent style is passed to its child unless you style the children differently!